src/frontend/src/common/utils/TreeModal/ConfirmButton/index.tsx

```diff
@@ -74,7 +75,7 @@ export const createConfirmButton = (treeId: number): FC => {
 
 async function getTreeUrl(treeId: number) {
   const result = await fetch(
-    `${process.env.API_URL}/api/auspice/view/${treeId}`,
+    `${envVariables.API_URL}/api/auspice/view/${treeId}`,
```

Comment:
use `envVariables.js` to access `process.env` variables, so we have single source of truth and know what variables are available to use!